src/types.ts

```diff
@@ -1,5 +1,75 @@
-import { FunctionInput } from "@azure/functions";
+import {
+    FunctionInput,
+    FunctionHandler,
+    InvocationContext,
+    FunctionResult,
+    FunctionOutput,
+    FunctionTrigger,
+} from "@azure/functions";
+import { DurableOrchestrationClient, EntityState } from "../src/classes";
+import { DurableEntityBindingInfo } from "../src/durableentitybindinginfo";
+import { IEntityFunctionContext } from "../src/ientityfunctioncontext";
+import { IOrchestrationFunctionContext } from "../src/iorchestrationfunctioncontext";
+import { OrchestratorState } from "../src/orchestratorstate";
+import { DurableOrchestrationInput } from "../src/testingUtils";
 
+// orchestrations
+export type OrchestrationHandler = (
+    context: IOrchestrationFunctionContext
+) => Generator<unknown, unknown, any>;
+
+export type OrchestrationFunction = FunctionHandler &
+    ((
+        context: IOrchestrationFunctionContext,
+        orchestrationTrigger: DurableOrchestrationInput
+    ) => Promise<OrchestratorState>);
+
```

Comment:
We have an `OrchestrationHandler` and an `OrchestrationFunction` because we make a difference between `df.orchestration` and `df.createOrchestration`, right? If we only had one of these methods, would we also only need one of these types?